src/Modules/Profile/Commands.Profile/GetPowerBIAccessToken.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Text;
+using Microsoft.PowerBI.Commands.Common;
+using Microsoft.PowerBI.Common.Abstractions.Interfaces;
+
+namespace Microsoft.PowerBI.Commands.Profile
+{
+    [Cmdlet(CmdletVerb, CmdletName)]
+    [OutputType(typeof(Hashtable), typeof(string))]
+    public class GetPowerBIAccessToken : PowerBICmdlet
```

Comment:
Are we able to relatively easily add E2E tests for this cmdlet?

Reply:
With some refactoring I think so.